worldedit-core/doctools/build.gradle.kts

```diff
@@ -1,14 +1,14 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-    kotlin("jvm") version "1.4.21"
+    kotlin("jvm") version "1.5.0-RC"
     application
 }
 
 applyCommonConfiguration()
 
 tasks.withType<KotlinCompile> {
-    kotlinOptions.jvmTarget = "1.8"
+    kotlinOptions.jvmTarget = "1.11"
```

Comment:
Wouldn't this be 11, as of Java 9 versions are no longer indexed with `1.`

Reply:
I'll double check this